aries-site/src/data/structures/templates/templates.js

```diff
@@ -244,6 +244,20 @@ export const templates = [
     sections: [],
     tags: [],
   },
+  {
+    name: 'Progressbar',
+    available: true,
+    cardOrder: 5,
+    description: 'a visual indicator of progress in a process or journey.',
```

Comment:
A progress bar is a visual indicator that displays the total amount of completion for a process, allowing users to gauge how much of a process is complete and how much remains.